util/weekly_build/SpackStackBuildCache_AllSteps.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+echo "build host: $(hostname)"
+
+set -exa
+
+cd $(dirname $0)
+
+. ShellSetup.sh $*
+export SETUPDONE=YES
+
+function trap_and_run {
+  trap ERROR ERR
+  scriptname=$1
+  function ERROR { alert_cmd $scriptname ; exit 1;}
+  $*
+}
+
+trap_and_run ./1_DirectorySetup.sh $*
```

Comment:
All these files should have two-digit numbers: 01_DirectorySetup.sh etc

Reply:
Fine with me, I'm curious why though?

Reply:
I see more than 9 of those in the future, and I would prefer to see them ordered correctly in bash etc